assignment2/inheritance.js

```diff
@@ -0,0 +1,21 @@
+function employee (nameS)
+{
+    this.name=nameS;
+    
+}
+//employee.prototype.getname=function()
+//
+ //   return this.name;
+//}
+function manager(salaryS){
```

Comment:
@psinha20 Please use space after the `()` and also when assigning. so, `this.salary=salaryS` should be 'this.salary = salaryS;'